.travis.yml

```diff
@@ -1,18 +1,22 @@
 language: php
 
-php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
-    - 7.0
-    - hhvm
-
 matrix:
     fast_finish: true
     include:
         - php: 5.3
           env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+        - php: 5.3
+          env: 'COMPOSER_FLAGS="--prefer-stable"'
+        - php: 5.4
+          env: 'COMPOSER_FLAGS="--prefer-stable"'
+        - php: 5.5
+          env: 'COMPOSER_FLAGS=""'
```

Comment:
this can be omitted (bash is fine with env variables which don't exist, and will give you an empty string)